charts/velero/crds/backups.yaml

```diff
@@ -8,7 +8,6 @@ metadata:
     "helm.sh/hook": pre-install,pre-upgrade
     "helm.sh/hook-delete-policy": "before-hook-creation"
     controller-gen.kubebuilder.io/version: v0.2.4
```

Comment:
I think we should change the `controller-gen.kubebuilder.io/version` to `v0.3.0`, same as velero v1.5.1 release [1].